include/seqan3/io/alignment_file/output.hpp

```diff
@@ -829,61 +833,56 @@ class alignment_file_output
 /*!\brief Deduces selected_field_ids from input and sets alignment_file_output::ref_ids_type to
  * seqan3::detail::ref_info_not_given. Valid formats and stream_char_type are defaulted.
```

Comment:
You forgot to adapt the documentation in this file

Reply:
good catch